tests/Resources/Page/ODH/JupyterHub/LaunchJupyterHub.robot

```diff
@@ -15,4 +15,5 @@ Launch Jupyterhub
    Input Text  xpath://input[@data-test-id="item-filter"]  jupyterhub
    Wait Until Page Contains  jupyterhub  timeout=15
    Click Element  partial link:https://jupyterhub
+   Sleep  10s
```

Comment:
prefer some wait for element, but I think you mentioned it was not very clear here.